src/subshell/common.c

```diff
@@ -465,7 +465,7 @@ init_subshell_child (const char *pty_name)
 
     case SHELL_FISH:
         execl (mc_global.shell->path, mc_global.shell->path, "--init-command",
-               "set --global __mc_csi_u 1", (char *) NULL);
+               "set --global __mc_kitty_keyboard 1", (char *) NULL);
```

Comment:
Since you care about commit messages, in 5dc7aefcae8514cfda1791b0f2abb7c23b3621f4:

> I guess we could avoid that but I don't think this
> is **not** a big loss, especially because since fish 4.0.1 will enable it again.

The double negation is wrong.